set.mm

```diff
@@ -20228,13 +20228,26 @@ Corresponds to the dual of Axiom (B) of modal logic.  (Contributed by NM,
   $}
 
   ${
-    $d x z $.  $d y z $.
+    $d x z w $.  $d y z w $.  $d ph w $.
     hbsbw.1 $e |- ( ph -> A. z ph ) $.
     $( If ` z ` is not free in ` ph ` , it is not free in ` [ y / x ] ph ` when
        ` y ` and ` z ` are distinct.  Version of ~ hbsb with a disjoint
-       variable condition, which does not require ~ ax-13 .  (Contributed by
-       NM, 12-Aug-1993.)  (Revised by Gino Giotto, 10-Jan-2024.) $)
+       variable condition, which requires less axioms.  (Contributed by NM,
```

Comment:
"fewer axioms"
Can you move this theorem before ax-12 ? (and similarly if this applies to other theorems below)

Reply:
Done. The other theorems either have ax-12 or ax-ext, so they cannot be moved earlier.